src/js/main.js

```diff
@@ -20,6 +20,14 @@ import { checkForConfiguratorUpdates } from './utils/checkForConfiguratorUpdates
 import * as THREE from 'three';
 import * as d3 from 'd3';
 
+if (/android/i.test(navigator.userAgent)) {
+    if (typeof String.prototype.replaceAll === "undefined") {
+        String.prototype.replaceAll = function(match, replace) {
+            return this.replace(new RegExp(match, 'g'), () => replace);
+        };
+    }
+}
```

Comment:
I don't think we need that android test tbh. `typeof` sort of covers all the required bases imo.

Reply:
thank you; sounds fair. if that's the case @haslinghuis can revert and it can just be merged since it was tested the other way already,